plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -561,7 +573,24 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/cloud/users/" \
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
 
+#
```

Comment:
This should be fixed by following https://github.com/coreruleset/nextcloud-rule-exclusions-plugin/issues/2#issuecomment-1304574373, I believe.

Reply:
Why isn't the default HTTP policy set in the official plugin?

Reply:
@GenialHacker Because plugins are a new feature which still has some limitations and things that needs to be resolved. This wasn't a case before, when exclusion rule packages were a part of core CRS.